sdk/nodejs/randomPassword.ts

```diff
@@ -16,6 +16,28 @@ import * as utilities from "./utilities";
  * 
  * This resource *does* use a cryptographic random number generator.
  * 
+ * ## Example Usage
+ * 
+ * 
+ * 
+ * ```typescript
+ * import * as pulumi from "@pulumi/pulumi";
+ * import * as aws from "@pulumi/aws";
+ * import * as random from "@pulumi/random";
+ * 
+ * const password = new random.RandomPassword("password", {
+ *     length: 16,
+ *     special: true,
+ *     overrideSpecial: `_%@`,
+ * });
+ * const example = new aws.rds.Instance("example", {
+ *     instanceClass: "db.t3.micro",
+ *     allocatedStorage: 64,
+ *     engine: "mysql",
+ *     username: "someone",
+ *     password: random_string.password.result,
```

Comment:
@stack72 this example is buggy. Can we fix it in our fork? This should be `password.result`, but isn't b/c (I assume) the original example is using `random_string.password.result` instead of `random_password.password.result`.